.github/workflows/build.yml

```diff
@@ -0,0 +1,62 @@
+name: Build
+
+on:
+  push:
+    branches:
+      - main
+      - develop
```

Comment:
We will probably not have a develop branch since I'm opting for using cactus model as a branch strategy.

Also, this rule here "push" -> would be good to set up master as protected branch, so no direct pushes to master will be allowed.

Also It would be very good to run CI (tests) on master AFTER merge to master.